tasks/broad/JointGenotypingTasks.wdl

```diff
@@ -884,40 +882,40 @@ task CrossCheckFingerprint {
     cp $vcfInputsList vcf_inputs.list
 
     java -Dpicard.useLegacyParser=false -Xms~{memMb - 512}m \
-      -jar /usr/gitc/PicardPublicWithCrosscheckNIOandSampleMapping.jar \
-      CrosscheckFingerprints \
-      --INPUT gvcf_inputs.list \
-      --SECOND_INPUT vcf_inputs.list \
-      --HAPLOTYPE_MAP ~{haplotype_database} \
-      --INPUT_SAMPLE_FILE_MAP ~{sample_name_map} \
-      --CROSSCHECK_BY SAMPLE \
-      --CROSSCHECK_MODE CHECK_SAME_SAMPLE \
-      --NUM_THREADS ~{cpu} \
-      --SKIP_INPUT_READABLITY_TEST \
-      ~{true='--EXIT_CODE_WHEN_MISMATCH 0' false='' scattered} \
-      --OUTPUT ~{output_name}
+    -jar /usr/gitc/PicardPublicWithCrosscheckNIOandSampleMapping.jar \
+    CrosscheckFingerprints \
+    --INPUT gvcf_inputs.list \
+    --SECOND_INPUT vcf_inputs.list \
+    --HAPLOTYPE_MAP ~{haplotype_database} \
+    --INPUT_SAMPLE_FILE_MAP ~{sample_name_map} \
+    --CROSSCHECK_BY SAMPLE \
+    --CROSSCHECK_MODE CHECK_SAME_SAMPLE \
+    --NUM_THREADS ~{cpu} \
+    --SKIP_INPUT_READABLITY_TEST \
```

Comment:
We can run this from the GATK jar if we comment out the --SKIP_INPUT_READABLITY_TEST  arg:
    gatk --java-options -Xms~{memMb - 512}m \
      CrosscheckFingerprints \
      --INPUT gvcf_inputs.list \
      --SECOND_INPUT ~{vcf_path} \
      --HAPLOTYPE_MAP ~{haplotype_database} \
      --INPUT_SAMPLE_FILE_MAP ~{sample_name_map} \
      --CROSSCHECK_BY SAMPLE \
      --CROSSCHECK_MODE CHECK_SAME_SAMPLE \
      --NUM_THREADS ~{cpu} \
      ~{true='--EXIT_CODE_WHEN_MISMATCH 0' false='' scattered} \
      --OUTPUT ~{output_name}

I don't want to run off this branch anymore because the line numbers from exceptions don't jive with master.

Reply:
Okay, I've updated this to use gatk now.